website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -47,9 +54,22 @@ To cancel a scheduled auto destroy run in Terraform Cloud:
 1. Under **Automatically destroy**, click **Edit** next to your scheduled run's details.
 1. Click **Remove**.
 
-You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto destroy run, and to report auto destroy run results.
+#### Destroy if Inactive
+
+You can configure Terraform Cloud to automatically destroy a workspace's infrastructure after a period of inactivity. Any runs that update Terraform state will push back the scheduled auto destroy time by the configured activity duration designated by the user.
+
+To schedule an auto destroy run after a period of inactivity in Terraform Cloud:
+1. Navigate to the workspace's **Settings** > **Destruction and Deletion** page.
+1. Under **Automatically destroy**, click **Set up auto destroy**.
+1. Click the **Destroy if inactive** toggle.
+1. Enter the desired activity duration.
+1. Click **Confirm auto destroy**.
+
+When configured for the first time, the auto destroy duration setting will display a date and time on which an auto destroy run is scheduled to be performed.
+This auto destroy date will be updated upon subsequent runs.
+Each run that updates state will push out the scheduled auto destroy time by the length of time configured in the duration setting.
```

Comment:
I think we can remove this sentence because you say this in the above paragraph:
```suggestion
```